src/features/staging.ts

```diff
@@ -84,13 +114,26 @@ export const StagingMixin = (ModelViewerElement:
         [$tick](time: number, delta: number) {
           super[$tick](time, delta);
 
-          if (this.autoRotate) {
-            (this as any)[$scene].pivot.rotation.y +=
-                ROTATION_SPEED * delta * 0.001;
+          if (!this.autoRotate) {
+            return;
+          }
+
+          this[$autoRotateTimer].tick(delta);
+
+          if (this[$autoRotateTimer].hasStopped) {
+            (this as any)[$scene].pivot.rotation.y += ROTATION_SPEED * delta * 0.001;
```

Comment:
In a subsequent PR I'll explore what it would be like to ease the rotation based on the rotation timer. This is why I added `Timer.timeScale`.

Reply:
👍